Apps/W1/EDocumentConnectors/Tietoevry/app/src/Extensions/EDocFormat.EnumExt.al

```diff
@@ -0,0 +1,15 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.EServices.EDocumentConnector.Tietoevry;
+
+using Microsoft.EServices.EDocument;
+
+enumextension 6390 "E-Doc. Format Ext." extends "E-Document Format"
```

Comment:
We should be able to remove this now if not mistaken.

Reply:
I can't see how this can be removed. If removed, I cannot set the correct format for the connector.

Reply:
Check the already shipped connector format. Should be the same as you have.

Reply:
We still have som custom code taking place in Interface procedure "Create". Populate some metafields in EDocument. No

Reply:
Resolved.